examples/all-clusters-app/linux/main-common.cpp

```diff
@@ -50,297 +50,43 @@
 
 #include <Options.h>
 
+#define CHIP_EVENT_COMMAND_SIZE 80
+
+#define CHIP_EVENT_FIFO_PATH                                                                                                       \
```

Comment:
could this be C++: `static constexpr const char kChipEventFifoPathPrefix[] = "/tmp/chip_all_cluster_fifo"`?